docs/_includes/calendar.html

```diff
@@ -0,0 +1,223 @@
+<div id="calendar">
+    {% include section-header.html name="calendar" version=page.versions.calendar %}
+
+    <p>The calendar component is used to display months, dates, and date ranges.</p>
+
+    <p>The base building block of a calendar is a table which contains dates for a single month. This can be either static or interactive, and looks the same for all types of calendars.</p>
+
+    <h4>Static Month</h4>
```

Comment:
Instead of these static month and dynamic code block examples, can we just start the docs with the default calendar? The table isn't meant to be portable outside of the `.calandar` class.

Reply:
I wasn't happy with this either, but since I reference these later and you can use both static and dynamic in every type of calendar I thought it was the most effective way of representing all of the information. I will play around with other ways of representing this using the default calendar instead.